test/bigtest/tests/loan-history/loan-history-form-test.js

```diff
@@ -121,18 +121,49 @@ describe('Loan History Form', () => {
       });
     });
 
-    describe('selecting any radio button and clicking save button', () => {
+    describe('selecting any radio button', () => {
       beforeEach(async () => {
         await LoanHistoryForm
           .feeFine.selectRadio(closingTypesMap.IMMEDIATELY)
           .feeFine.selectRadio(closingTypesMap.INTERVAL)
           .feeFine.selectRadio(closingTypesMap.NEVER);
+      });
 
-        await LoanHistoryForm.saveButton.click();
+      describe('when checkbox is unchecked', () => {
+        beforeEach(async () => {
+          await LoanHistoryForm.clickTreatEnabledCheckbox();
+        });
+
+        it('should disappear section for closed loans with associated fees/fines', () => {
+          expect(LoanHistoryForm.feefineSection.isPresent).to.be.false;
+        });
+
+        describe('when checkbox is checked', () => {
+          beforeEach(async () => {
+            await LoanHistoryForm.clickTreatEnabledCheckbox();
+          });
+
+          const check = items => {
```

Comment:
Could you please clarify if there are any reasons to declare a global function and use it only in one place?

I believe, we can try just to declare a constant isChecked and assign a result to it (Line 150-151), or add this util function to the interactor.

@SergiyVSergiyenko What do you think about it?